flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/table/JdbcRowDataLookupFunction.java

```diff
@@ -71,7 +75,9 @@ public JdbcRowDataLookupFunction(
             String[] fieldNames,
             DataType[] fieldTypes,
             String[] keyNames,
-            RowType rowType) {
+            RowType rowType,
```

Comment:
This looks strange: I wonder why do we need this if it is not used and at the same time we still `fieldNames` and `fieldTypes` containing same info

Reply:
yes, however it could be calculated from `fieldNames` and `fieldTypes` exactly in same way like it is done at lines 109-110
anyway this is just a thing to note, it seems not related to this PR

Reply:
Interesting - I see.